low_cost_robot_arm/low_cost_robot_arm.xml

```diff
@@ -0,0 +1,136 @@
+      <default class="visual">
+        <geom type="mesh" group="2" contype="0" conaffinity="0"/>
+      </default>
+      <default class="collision">
+        <geom type="mesh" group="3" mass="0" density="0"/>
+        <default class="pad_box1">
+          <geom type="box" friction="0.7" solimp="0.95 0.99 0.001" solref="0.004 1" mass="0" priority="1" size="0.002 0.002 0.0025" rgba="0.2 0.2 0.2 1"/>
+        </default>
+        <default class="pad_box2">
+          <geom type="box" friction="0.6" solimp="0.95 0.99 0.001" solref="0.004 1" mass="0" priority="1" size="0.002 0.002 0.0052" rgba="0.1 0.1 0.1 1"/>
+        </default>
+      </default>
+    </default>
+  </default>
+
+  <asset>
+    <material name="black" rgba="0.1 0.1 0.1 1" specular="0.1" shininess="0.1"/>
+    <material name="white" rgba="0.8 0.8 0.8 1" specular="0.1" shininess="0.1"/>
+
+    <mesh name="base_link" file="base_link.stl"/>
+    <mesh name="shoulder_rotation" file="shoulder_rotation.stl"/>
+    <mesh name="shoulder_to_elbow" file="shoulder_to_elbow.stl"/>
+    <mesh name="elbow_to_wrist_extension" file="elbow_to_wrist_extension.stl"/>
+    <mesh name="elbow_to_wrist" file="elbow_to_wrist.stl"/>
+    <mesh name="gripper_static_finger" file="gripper_static_finger.stl"/>
+    <mesh name="gripper_moving_finger" file="gripper_moving_finger.stl"/>
+
+    <mesh name="base_link_motor" file="base_link_motor.stl"/>
+    <mesh name="shoulder_rotation_motor" file="shoulder_rotation_motor.stl"/>
+    <mesh name="shoulder_to_elbow_motor" file="shoulder_to_elbow_motor.stl"/>
+    <mesh name="elbow_to_wrist_extension_motor" file="elbow_to_wrist_extension_motor.stl"/>
+    <mesh name="elbow_to_wrist_motor" file="elbow_to_wrist_motor.stl"/>
+    <mesh name="gripper_static_finger_motor" file="gripper_static_finger_motor.stl"/>
+
+    <mesh name="base_link_collision" file="base_link_collision.stl"/>
+    <mesh name="shoulder_rotation_collision" file="shoulder_rotation_collision.stl"/>
+    <mesh name="shoulder_to_elbow_collision" file="shoulder_to_elbow_collision.stl"/>
+    <mesh name="elbow_to_wrist_extension_collision" file="elbow_to_wrist_extension_collision.stl"/>
+    <mesh name="elbow_to_wrist_collision" file="elbow_to_wrist_collision.stl"/>
+    <mesh name="gripper_static_finger_collision_1" file="gripper_static_finger_collision_1.stl"/>
+    <mesh name="gripper_static_finger_collision_2" file="gripper_static_finger_collision_2.stl"/>
+    <mesh name="gripper_moving_finger_collision_1" file="gripper_moving_finger_collision_1.stl"/>
+    <mesh name="gripper_moving_finger_collision_2" file="gripper_moving_finger_collision_2.stl"/>
+
+  </asset>
+
+  <worldbody>
+    <body name="base_link" childclass="low_cost_robot_arm" quat="-0.707 0. 0. 0.707">
+      <geom mesh="base_link" class="visual" material="white"/>
+      <geom mesh="base_link_motor" class="visual" material="black"/>
+      <geom mesh="base_link_collision" class="collision"/>
+      
```

Comment:
Can you remove all newlines between the bodies? In general, it would be great if you could follow our [XML guide](https://github.com/google-deepmind/mujoco_menagerie/blob/main/CONTRIBUTING.md#xml-style), in particular using the auto formatter.